src/config.rs

```diff
@@ -157,22 +170,24 @@ impl ConfigBuilder {
         self.additional_fields.extend(additional_fields);
         self
     }
+
     /// Set the policy to apply when async send buffer is full.
     ///
     /// It is recommended to use the `FullBufferPolicy::Discard` policy.
     ///
-    /// If not set or set to `None`, `FullBufferPolicy::Discard` will be used by default
+    /// If not set or set to `None`, `FullBufferPolicy::Discard` will be used by
+    /// default
     pub fn set_full_buffer_policy(mut self, policy: Option<FullBufferPolicy>) -> ConfigBuilder {
         self.full_buffer_policy = policy;
         self
     }
     /// Set the TCP connect timeout.    
-    pub fn set_connect_timeout_ms(mut self, connect_timeout_ms: Option<u64>) -> ConfigBuilder {
+    pub fn set_connect_timeout_ms(mut self, connect_timeout_ms: Option<Duration>) -> ConfigBuilder {
         self.connect_timeout_ms = connect_timeout_ms;
         self
     }
     /// Set the TCP write timeout.    
-    pub fn set_write_timeout_ms(mut self, write_timeout_ms: Option<u64>) -> ConfigBuilder {
+    pub fn set_write_timeout_ms(mut self, write_timeout_ms: Option<Duration>) -> ConfigBuilder {
```

Comment:
```suggestion
    pub fn set_write_timeout(mut self, write_timeout_ms: Option<Duration>) -> ConfigBuilder {
```
Let's remove all `_ms` suffixes where the duration was moved from `u64` to `std::time::Duration`.